src/sim-host/ui/custom-elements.js

```diff
@@ -412,6 +480,14 @@ function registerCustomElement(name) {
     });
 }
 
+function getId() {
+    var result = [];
+    for (var i = 0; i < 8; i++) {
+        result.push((Math.random() * 16 | 0).toString(16));
```

Comment:
Oh hey, in HTML5 ids are super broad and can start with numbers, so feel free to ignore the second point there.